charts/externalip-webhook-1.0.0/templates/webhook.yaml

```diff
@@ -23,5 +23,6 @@ webhooks:
     - UPDATE
     resources:
     - services
-  {{ .SideEffects }}
-  {{ .AdmissionReviewVersions }}
+  sideEffects: None
+  admissionReviewVersions:
```

Comment:
why do we need to support v1beta1?

Reply:
That's what's in the upstream config: https://github.com/kubernetes-sigs/externalip-webhook/blob/master/config/webhook/manifests.yaml

This webhook hasn't been meaningfully updated in years.  It was essentially done day one.